src/App.tsx

```diff
@@ -15,48 +81,122 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+function orderedGoods(
+  goods: string[],
+  isReversed: boolean,
+  sortType: SortType,
+) {
+  const goodsOnPage = [...goods];
+
+  goodsOnPage.sort((first, second) => {
+    switch (sortType) {
+      case SortType.ALPHABET:
+        return first.localeCompare(second);
+
+      case SortType.LENGTH:
+        return first.length - second.length;
+
+      default:
+        return 0;
```

Comment:
```suggestion

     case SortType.NONE:
      default:
        return 0;
```

best practice is to handle all cases